src/main/java/seedu/address/ui/TagCard.java

```diff
@@ -43,6 +45,12 @@ public TagCard(Tag tag, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         tagName.setText(tag.getTagName().tagName);
         fileAddress.setText(tag.getFileAddress().value);
+        for (seedu.address.model.label.Label label : tag.getLabels()) {
+            String labelText = " " + label.getLabel() + " ";
```

Comment:
Is it better to declare a static string and use string.format in here? I feel like it is better to use it rather than a magic string even though it is only " ".

Reply:
emmm, ya, I agree it's not very good to use “ ” directly, but I think it's quite simple to use that here.

Reply:
Alright!